frontend/src/components/print/config/PicassoConfig.vue

```diff
@@ -25,16 +38,7 @@ export default {
   },
   data() {
     return {
-      orientations: [
-        {
-          value: 'L',
-          text: 'Landscape',
-        },
-        {
-          value: 'P',
-          text: 'Portrait',
-        },
-      ],
+      orientations: ORIENTATIONS,
```

Comment:
That the const cannot be changed, you could use cloneDeep

Reply:
https://github.com/ecamp/ecamp3/pull/4559/commits/2350df1fa59e7824afd84f67510e0be3679b6651